bcda/responseutils/writer.go

```diff
@@ -117,10 +117,26 @@ func CreateCapabilityStatement(reldate time.Time, relversion, baseurl string) *f
 			},
 		},
 	}
-
+	checkEnv(statement, baseurl)
 	return statement
 }
 
+func checkEnv(statement *fhirmodels.CapabilityStatement, baseUrl string) {
+	if os.Getenv("ENABLE_PATIENT_EXPORT") == "true" {
+		restComponent := statement.Rest[0].Operation
+
+		element := fhirmodels.CapabilityStatementRestOperationComponent{
+			Name: "export",
```

Comment:
I don't see any testing for this.  If we don't have any tests for the capability statement already I would love to see a ticket for adding them.

Reply:
Ticket created for our unit tests capability statement here: https://jira.cms.gov/browse/BCDA-658. But I do think this is outside the scope of this particular PR since it's covering the entire metadata endpoint and not just this additional piece.